### ISSUE TITLE
Minor compile issues on Cygwin

### ISSUE THREAD

**Issue by dziegel** [appneta/tcpreplay#412]
tcpreplay-4.2.5.tar.gz on cygwin:

1) Instructions to compile are wrong on website in "Special Instructions for Windows" section - one should unpack winpcap to /wpdpack and simply run ./configure. Using "--with-libpcap=..." breaks the configure run, it won't detect libpcap.

2) sendpacket.h needs to be changed not to #include <netinet/if_ether.h>, e.g.:
```
#ifdef __NetBSD__
#include <net/if_ether.h>
#else
#ifndef __CYGWIN__
#include <netinet/if_ether.h>
#endif
#endif
```

Apart from this, compiling it works with one warning in GCC6.3:
```
In file included from libopts.c:31:0:
nested.c: In function 'optionLoadNested':
nested.c:789:19: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
                   if (*text == ',') text++; break;
                   ^~
nested.c:789:45: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'if'
                   if (*text == ',') text++; break;
                                             ^~~~~
```

and tests are OK:
```
$ make test
echo Making test in ./test
Making test in ./test
cd ./test && make test
make[1]: Entering directory 'C:/cygwin/tcpreplay-4.2.5/test'
rm: das Entfernen von 'test.log' ist nicht möglich: No such
NOTICE: Tests must be run as root
Sending traffic on '%0' and '%0'
[tcpprep] Auto/Router mode test:                OK
[tcpprep] Auto/Bridge mode test:                OK
[tcpprep] Auto/Client mode test:                OK
[tcpprep] Auto/Server mode test:                OK
[tcpprep] Auto/First mode test:                 OK
[tcpprep] CIDR mode test:                       OK
[tcpprep] Regex mode test:                      OK
[tcpprep] Port mode test:                       OK
[tcpprep] MAC mode test:                        OK
[tcpprep] Comment mode test:                    OK
[tcpprep] Print info mode test:                 OK
[tcpprep] Print comment mode test:              OK
[tcpprep] Config mode test:                     OK
[tcpprep] MAC reverse mode test:                OK
[tcpprep] CIDR reverse mode test:               OK
[tcpprep] Regex reverse mode test:              OK
[tcpreplay] Basic test:                         OK
[tcpreplay] Cache test:                         OK
[tcpreplay] Packets/sec test:                   OK
[tcpreplay] Mbps test:                          OK
[tcpreplay] Topspeed test:                      OK
[tcpreplay] Config file/VLAN add test:          OK
[tcpreplay] Multiplier test:                    OK
[tcpreplay] Packets/sec Multiplier test:        OK
[tcpreplay] Precache test:                      OK
[tcpreplay] Statistics test:                    OK
[tcpreplay] Dual file test:                     OK
[tcpreplay] Maximum sleep test:                 OK
[tcprewrite] Portmap test:                      OK
[tcprewrite] Endpoint test:                     OK
[tcprewrite] Pseudo NAT test:                   OK
[tcprewrite] Truncate test:                     OK
[tcprewrite] Pad test:                          OK
[tcprewrite] Seed IP test:                      OK
[tcprewrite] Src/Dst MAC test:                  OK
[tcprewrite] Layer2 test:                       OK
[tcprewrite] Config/VLAN Add test:              OK
[tcprewrite] Skip bcast test:                   OK
[tcprewrite] DLT User test:                     OK
[tcprewrite] DLT Cisco HDLC test:               OK
[tcprewrite] VLAN Delete test:                  OK
[tcprewrite] Remove EFCS:                       OK
[tcprewrite] Force TTL:                         OK
[tcprewrite] Increase TTL:                      OK
[tcprewrite] Reduce TTL:                        OK
[tcprewrite] TOS test:                          OK
[tcprewrite] MTU Truncate test:                 OK
[tcprewrite] Substitute Src/Dst MAC test:       OK
[tcprewrite] Seeded MAC test:                   OK
[tcprewrite] Seeded Keep MAC test:              OK
[tcprewrite] L7 fuzzing test:                   OK
make[1]: Leaving directory 'C:/cygwin/tcpreplay-4.2.5/test'
```

**Comment by fklassen** [comment]
Thanks for the bug report. Scheduled for next release.

**Comment by fklassen** [comment]
The first issue isn't a problem me. In some cases I have heard that users must add `--enable-static-link`, e.g.

    ./configure --with-libpcap=/home/fklassen/git/libpcap --enable-static-link

Second issue fixed with PR #463
